### PR TITLE
tighten up styles/add more dark mode support

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5223,7 +5223,8 @@ libra-docusaurus-nav@^1.2.6:
   integrity sha512-3/E2w9dMfRAxBP9rqguuq6R1O/2IC8eL9GDzBegcWyR6MBkOcLI8y6gEJ+RXVvllY8Afg6R7lAW0m1ZSdbIWwQ==
   dependencies:
     prop-types "^15.7.2"
-    react "^16.13.1"
+    react "^16.8.4"
```
Comment (jjwill) [inline yarn.lock:5226]: Why did we decrement the version?

Comment (oriooctopus) [reply yarn.lock:5226]: @jjwill We didn't, we incremented it

Comment (jjwill) [reply yarn.lock:5226]: We went from a newer one to an older one. 16.8.4 is older than 16.13.1. I might be missing something here though

Comment (oriooctopus) [reply yarn.lock:5226]: Oh lol, I was thinking 80 for some reason

**Pull request merged**